### DISCUSSION TITLE
Local Backup + Cron

### DISCUSSION THREAD

**Discussion by ghost** [Kovah/LinkAce#850] [Support]
In the 1.15 version (advance) it is impossible for me to set up local_backups.
I can't get the .zip file in the /app/storage/app/backups folder of my host even with all the info in the .env file.
The cron task appears in the logs but nothing more. 
Still got the issue on a clean docker with full permissions on the folder in debian.
Can anyone help me with that ?

**Comment by zarevskaya** [comment]
> In the 1.15 version (advance) it is impossible for me to set up local_backups.
> I can't get the .zip file in the /app/storage/app/backups folder of my host even with all the info in the .env file.
> The cron task appears in the logs but nothing more. 
> Still got the issue on a clean docker with full permissions on the folder in debian.
> Can anyone help me with that ? 

I have the same issue. I also tested various permissions, nothing works.

Any hint? Thanks!

**Reply by Kovah** [reply]
I will have a look soon to get this properly working with v2.

**Reply by Kovah** [reply]
I am currently testing backups for version 2. Will report back with the results soon.

**Reply by Kovah** [reply]
I noticed that whatever is configured inside the container, Docker Compose will use the outside folder as a reference and may break permissions.

- non-working setup:
  - Outside folder `./backups` has permissions 0755.
  - Inside folder `/app/storage/app/backups` gets permissions 0755.
  - Backups fail with a permission issue (not writable)
- working setup:
  - Outside folder `./backups` has permissions 0766 via `chmod 0766 ./backups`
  - Inside folder `/app/storage/app/backups` gets permissions 0766.
  - Backups run through without issues.

I have updated the docs for v2 accordingly.
